.appveyor.yml

```diff
@@ -13,7 +13,7 @@ install:
   - ps: Install-Product node 8
   - chocolatey install make
   - chocolatey install gnuwin32-coreutils.portable
-  - npm install
+  - appveyor-retry npm install
```

Comment:
This is great. Good find. 
Can (should) we set the number of retries at all?

Reply:
I don't think so.